src/replicache.test.ts

```diff
@@ -1616,6 +1616,84 @@ testWithBothStores('HTTP status push', async () => {
   expect(okCalled).to.equal(true);
 });
 
+testWithBothStores('poke', async () => {
+  // Things to test:
```

Comment:
is this a todo or is it explaining what this is testing? Please rephrase to make it clear

Reply:
These are TODOs but I'm still trying to understand the specific cases.